tests/microos/disk_boot.pm

```diff
@@ -20,7 +20,7 @@ sub run {
     # default timeout in grub2 is set to 10s
     # Sometimes, machines tend to stall when trying to match grub2
     # this leads to test failures because openQA does not assert grub2 properly
-    if (is_aarch64 || is_sle_micro('>=6.0')) {
+    if ((is_aarch64 || is_sle_micro('>=6.0') && !main_micro_alp::is_dvd()) {
```

Comment:
This cannot be the fix for a failure on SLEM 5 because of `is_sle_micro('>=6.0')`.

Reply:
in the case of 5.5 DVD flavor, the last condition will be false and it will execute `shift->wait_boot(bootloader_time => 300);` which is what we need.

Reply:
Then the brackets are IMHO wrong. `&&` > `||`. In the aarch64 case, the if block will always be executed

Reply:
With the updated brackets it could work, yes. Please provide at least one VR.